tfx/components/example_gen/base_example_gen_executor.py

```diff
@@ -109,14 +125,20 @@ def _MaybeSerialize(x):
 
 @beam.ptransform_fn
 @beam.typehints.with_input_types(beam.Pipeline)
-@beam.typehints.with_output_types(Union[tf.train.Example, bytes])
+@beam.typehints.with_output_types(Union[tf.train.Example, 
+                                        tf.train.SequenceExample, 
+                                        bytes])
 def _InputToExampleOrBytes(
     pipeline: beam.Pipeline,
     input_to_example: beam.PTransform,
     exec_properties: Dict[Text, Any],
     split_pattern: Text,
 ) -> beam.pvalue.PCollection:
-  """Converts input into a tf.train.Example, or a bytes (serialized proto)."""
+  """Converts input into one of the following:
+     - tf.train.Example,
+     - tf.train.SequenceExample, 
+     - a bytes (serialized proto).
```

Comment:
I feel you might need a full doc (includes args and returns, check examples in this repo) if it's not a single line, otherwise line won't pass

Reply:
yes, I changed this back to one line.